src/neuroconv/datainterfaces/ecephys/intan/intandatainterface.py

```diff
@@ -95,6 +68,12 @@ def __init__(
         es_key : str, default: "ElectricalSeries"
         """
 
+        if stream_id is not None:
+            warnings.warn(
+                "Use of the 'stream_id' parameter is deprecated and it will be removed completly by September 2024..",
```

Comment:
```suggestion
                "Use of the 'stream_id' parameter is deprecated and it will be removed after September 2024.",
```

Better to leave loose lower bound rather than right upper bound on these things since sometimes exact release schedules are outside our control

Reply:
That makes sense.